sdk/core/System.BinaryData/src/System.BinaryData.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We will need to check-in https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/snk/Open.snk into our repo and use that strong name key for signing.